Source/Particles/Collision/CollisionHandler.cpp

```diff
@@ -95,12 +97,14 @@ void CollisionHandler::doCollisions ( int step, amrex::Real cur_time, amrex::Rea
 {
 
     if (m_use_global_debye_length) {
+        DOXY_GRAPH(MultiParticleContainer::GenerateGlobalDebyeLength();)
         mypc->GenerateGlobalDebyeLength();
     }
 
     for (auto& collision : allcollisions) {
         int const ndt = collision->get_ndt();
         if ( step % ndt == 0 ) {
+            DOXY_GRAPH(CollisionBase::doCollisions();)
             collision->doCollisions(cur_time, dt*ndt, mypc);
```

Comment:
This is an example of an approximate graph, where the type of `collision` is determined at runtime and cannot be analyzed statically. 

What I do here is display the call from the base virtual method, assuming that the developer who looks at the graph understands that the actual method called will be the one from the runtime derived class.